adapters/minutemedia/minutemedia.go

```diff
@@ -101,6 +101,8 @@ func extractOrg(openRTBRequest *openrtb2.BidRequest) (string, error) {
 			return "", fmt.Errorf("failed to unmarshal ImpExtMinuteMedia: %w", err)
 		}
 
+		// golangci-lint staticcheck SA4004: the surrounding loop is unconditionally terminated
+		// TODO: confirm this is intended and add '// nosemgrep: SA4004'
```

Comment:
Please resolve this TODO.

Reply:
I see it’s labelled as resolved, but the TODO comment is still there. Can you address this?

Reply:
done, added `//nolint: staticcheck // SA4004: the surrounding loop is unconditionally terminated`